One-page form revision (#23)
ezsystems/commerce-documentation: docs/guide/forms/forms.md

- After submitting some processes are executed in the backend.
- After server response the user sees a confirmation page with a success or error message.

eZ Commerce uses [Symfony forms](http://symfony.com/doc/3.4/forms.html) as a part of the solution.

You can pre-fill the form with default values when it is loaded for the first time.

> The process that pre-fills the form is called [`preDataProcessor`](form_api/predataprocessors.md).

Reviewer: I'd consider using the name "pre-data processor" nad not the actual name of the processor class here.

> The process that pre-fills the form is called [`preDataProcessor`](form_api/predataprocessors.md).

Author: ```suggestion
The process that pre-fills the form is called a [pre-data processor](form_api/predataprocessors.md).
```

You can define actions that are executed after the form is submitted.

> These actions are services that are called is a sequence and are able to exchange the data.

Reviewer: ```suggestion
These actions are services that are called in a sequence and are able to exchange the data.
```

"exchange" -> "replace" ?

            - ses_forms.send_confirmation_data_processor
```

|Configuration key|Description|
|--- |--- |

> |`modelClass`|Required. Fully-qualified class path to the form entity. This class must extend `Silversolutions\Bundle\EshopBundle\Entities\Forms\AbstractFormEntity`|

Reviewer: ```suggestion
|`modelClass`|Required. A fully-qualified class path to the form entity. This class must extend `Silversolutions\Bundle\EshopBundle\Entities\Forms\AbstractFormEntity`|
```

> |`typeClass`|Required if `typeService` is not defined. Fully-qualified class path to the form type. Using `typeService` gives more flexibility.|

Reviewer: ```suggestion
|`typeClass`|Required if `typeService` is not defined. A fully-qualified class path to the form type. Using `typeService` gives more flexibility.|
```
